docs/how-to-debug-onboarding-partner.md

```diff
@@ -0,0 +1,44 @@
+# How to debug Onboarding Partner
+
+On App's first launch, we try to detect the installation context so we can extract Onboarding Partner and display custom instructions to our users
+
+The Onboarding Partner feature is based on [Google Play Install Referrer API](https://developer.android.com/google/play/installreferrer)
+
+> **Note**
+> When an app is installed using a Referrer URI, the referrer value won't change until the App is reinstalled (or after 90days but our code will use the initial value anyway)
+
+## Build a Referrer URI
+
+In order to build a Rerrer URI, you can use the following generator
+- https://developers.google.com/analytics/devguides/collection/android/v4/campaigns#google-play-url-builder
+
+The expected values are the following:
+- Application ID: `io.cozy.flagship.mobile`
+- Campaign Source: the partner name
+- Campaign Content: the provided context (i.e `MesPapiers`, `default` etc)
+- Campaign Name: `onboarding_partner`
+
+## Testing on production (phone with Play Store installed)
+
+When testing on production, build a referrer URI and install the App by opening this URI from your device
+
+## Testing on local dev (phone with Play Store installed)
+
+If the feature you want to test is not deployed yet, the do the following:
```

Comment:
then do the

Reply:
Fixed in https://github.com/cozy/cozy-flagship-app/compare/e296534f408c5ca5dce5b8af44df9d6144165350..1b2e03a8f035457424a35b1ed39ad6fbb317bb73